packages/client-config/API.md

```diff
@@ -8,17 +8,17 @@ import { AwsCredentialIdentityProvider } from '@aws-sdk/types';
 import { DeployedBackendIdentifier } from '@aws-amplify/deployed-backend-client';
 
 // @public
-interface AmazonCognitoStandardAttributesConfig {
-    // (undocumented)
-    required?: boolean;
-}
+type AmazonCognitoStandardAttributes = 'address' | 'birthdate' | 'email' | 'family_name' | 'gender' | 'given_name' | 'locale' | 'middle_name' | 'name' | 'nickname' | 'phone_number' | 'picture' | 'preferred_username' | 'profile' | 'sub' | 'updated_at' | 'website' | 'zoneinfo';
 
-// @public (undocumented)
+// @public
 interface AmazonLocationServiceConfig {
     name?: string;
     style?: string;
 }
 
+// @public
+type AmazonPinpointChannels = 'IN_APP_MESSAGING' | 'FCM' | 'APNS' | 'EMAIL' | 'SMS';

```

Comment:
should we lower case these too ? and other places like that ?

Reply:
The cognito change I believe is influenced by the cognito service values (to avoid keep changing the case). These other constant values are owned by the amplify teams.

Reply:
To echo Praveen's note the Cognito change was to match the attribute names https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#cognito-user-pools-standard-attributes

Similar to Cognito, AppSync's auth types are derived from the service's values https://docs.aws.amazon.com/appsync/latest/devguide/security-authz.html#authorization-types

but yes for pinpoint these are constants owned by amplify teams (they're PascalCase `<Type>Channel` resources https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_pinpoint.CfnEmailChannel.html)

Reply:
We'll never achieve consistency :(